langlib/lang.float/src/main/ballerina/float.bal

```diff
@@ -33,6 +33,11 @@ public const float Infinity = 1.0/0.0;
 #
 # Exactly one of isFinite, isInfinite and IsNaN will be true for any float value
 #
+# ```ballerina
+# float finiteValue = 1.2;
+# boolean isFinite = finiteValue.isFinite();
```

Comment:
We generally use the `is` prefix with the function/method and not the variable. Let's discuss this and finalize a convention. This came up with the array langlib too.

Reply:
Shall I go with `finite`, `infinite` and `nan` as variable names for this scenarios instead of `isFinite`, `isInfinite`, `isnan`?
wdyt?

Reply:
+1

Reply:
Updated